perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -198,20 +198,35 @@ def ParseResults(ycsb_result_string, data_type='histogram'):
         indicates that 530 ops took between 0ms and 1ms, and 1 took between
         19ms and 20ms. Empty bins are not reported.
   """
+  lines = []
+  client_string = 'YCSB'
+  command_line = 'unknown'
   fp = io.BytesIO(ycsb_result_string)
-  client_string = next(fp).strip()
-  if not client_string.startswith('YCSB Client 0.'):
+  result_string = next(fp).strip()
+
+  def IsHeadOfResults(line):
+      return line.startswith('YCSB Client 0.') or line.startswith('[OVERALL]')
+
+  while not IsHeadOfResults(result_string):
+    result_string = next(fp).strip()
+
+  if result_string.startswith('YCSB Client 0.'):
+    client_string = result_string
+    command_line = next(fp).strip()
+    if not command_line.startswith('Command line:'):
+      raise IOError('Unexpected second line: {0}'.format(command_line))
+  elif result_string.startswith('[OVERALL]'):  # YCSB > 0.7.0.
```

Comment:
I wasn't sure changing to 0.9.0 would affect benchmarks other than Datastore and Bigtable, but it seems okay now cuz I went through the release changes and didn't see any changes about those databases.

But instead of installing the complete YCSB package, I would suggest just install the required binding for each benchmark. This means we might need some changes to `BaseLinuxMixin.Install`, so that we can specify the package URL when needed.